### PR TITLE
Add ardb defconfigs

### PR DESCRIPTION
**Pull request by MiloCasagrande**

### CODE REVIEW

File: jenkins/kernel-defconfig-creator.sh [hunk 1]
```diff
@@ -165,6 +165,15 @@ if [ ${TREE_NAME} = "lsk" ] || [ ${TREE_NAME} = "anders" ]; then
   fi
 fi
 
+# Ard specific tree and branch defconfigs.
```
Comment (mattface) [inline jenkins/kernel-defconfig-creator.sh:168]: I'm going to assume these are only wanted for ARM arch? How they are right now, they will be added to every architecture and will presumably fail on all the others.

**Review by mattface — CHANGES_REQUESTED** [review]
Your change will affect all architectures, and it looks like this duplicates the other pull request to update the pull url

Please do it to the kernelci-build-staging branch as commits here diverge the tree and become a p.i.t.a to deal with when making a pull request from staging, plus things like defconfig changes affect the whole system so i'd want to run at least some quick tests on it before merging to master.

**Comment by MiloCasagrande** [comment]
I thought better of github, that stacking changes on two different branches
would only show the latest one not both.

Why would it become a pita? I understand the testing part, but you can
always rebase (or squash) when you merge via github.
What I consider a pita, but I don't have to deal with the tree that often,
is having two separate trees. Why a "staging" branch wouldn't work in this
case?

-- 
Milo Casagrande
Linaro.org <www.linaro.org> │ Open source software for ARM SoCs

**Comment by MiloCasagrande** [comment]
Moving to staging.

**Pull request closed without merging**